src/Microsoft.DocAsCode.Dfm.VscPreview/DfmProcess.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace DfmProcess
+{
+    using System;
+    using System.IO;
+    using System.Text;
+
+    using Microsoft.DocAsCode.Dfm;
+    using Microsoft.DocAsCode.MarkdownLite;
+    using Microsoft.DocAsCode.Plugins;
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            try
+            {
+                while (true)
+                {
+                    string num_str = Console.ReadLine();
```

Comment:
get it, the process is listening the input

---
In reply to: [73845692](https://github.com/dotnet/docfx/pull/583#discussion_r73845692) [](ancestors = 73845692,73842828)